docs/content/mobile/android/local-deploy.md

```diff
@@ -1,73 +1,69 @@
 [//]: # (TODO - get rid of mobile_launch.json mention and explain it differently)
 
+# Prepare your project to connect to a local deployment
```

Comment:
this section should be below the "Before reading this document..." and there should only be one header 1 as that describes the overall intent of the doc.

So try to structure it like this:

```
 # Connecting to a local deployment

Before reading...

## Prepare your project to connect

...

## Connect to a local deployment

After everything has been set up, you have now three options on how to connect.

## Connecting your Android device to a local deployment using Unity Remote

..

## Connecting your Android emulator to a local deployment

..

## Connecting your Android device to a local deployment

..

```

Need to check how that would look, it might even be better to have expandable tabs for the three options.